test/UnitTestHelpers.hs

```diff
@@ -209,25 +255,6 @@ testActor = Actor
   , bproj = False
   }
 
-testItemKind :: ItemKind
```

Comment:
I see there were 4 different definitions of `testItemKind` and `emptyMultiGroupItem` that is preferable to them. You removed one of the 4 `testItemKind`. Is there a good reason to keep the others? If so, could you add comments before these definitions stating the reason?

Reply:
my error. They had slightly different values which resulted test failure. I slightly changed the `shared` testItemKind to satisfy the failing tests.

Reply:
And could `emptyMultiGroupItem` be used instead of `testItemKind`?

Reply:
yes, will do.  It will require modifying some of the tests

Reply:
Alternatively, perhaps define `testItemKind` as `emptyMultiGroupItem {asdf = sdfg, qwer = oiuou}` with a couple of fields changed `. And perhaps in some tests use `emptyMultiGroupItem` and in others `testItemKind` for better test coverage.